packages/eslint-config-algolia/rules/react.js

```diff
@@ -10,7 +10,112 @@ module.exports = {
         allowArrowFunctions: true,
       },
     ],
-    'react-hooks/rules-of-hooks': 'error',
-    'react-hooks/exhaustive-deps': 'warn',
+    'react-hooks/rules-of-hooks': ['error'],
+    'react-hooks/exhaustive-deps': ['warn'],
+
+    'react/button-has-type': ['error'],
+    'react/destructuring-assignment': ['off'],
+    'react/jsx-boolean-value': ['error', 'always'],
+    'react/jsx-filename-extension': ['error', { extensions: ['.tsx', '.jsx'] }],
+    'react/jsx-pascal-case': ['error'],
+    'react/jsx-no-target-blank': ['error'],
+    'react/jsx-wrap-multilines': [
```

Comment:
doesn't prettier already do this?

Reply:
mmh I'm not sure with this level of details